src/components/Canvas/FallingBall.jsx

```diff
@@ -35,12 +35,13 @@ function drawFallingBall (canvas) {
     }
   }
 
+  /** @type {import('./CanvasBuilder').DrawFunction} */
   function draw () {
     ctx.clearRect(0, 0, canvas.width, canvas.height)
 
     ball.update()
     ball.create()
-    requestId = requestAnimationFrame(draw)
+    requestId = requestAnimationFrame(() => draw())
```

Comment:
It's not as pretty, but it's more correct. `draw` expects an object of `props` and the `DOMHighResTimeStamp` is not type-compatible with `props`.